config/docker/fragment/api.jinja2

```diff
@@ -18,7 +18,7 @@ RUN git clone --depth=1 $api_url /tmp/kernelci-api
 WORKDIR /tmp/kernelci-api
 RUN git fetch origin $api_rev
 RUN git checkout FETCH_HEAD
-RUN python3 -m pip install .
+RUN python3 -m pip install .[dev]
```

Comment:
```suggestion
RUN python3 -m pip install '.[dev]'
```

Reply:
Ah, this needs a rebase. I'll do it in a minute.